node/src/components/consensus/highway_core/state.rs

```diff
@@ -429,6 +429,8 @@ impl<C: Context> State<C> {
             }
         };
         self.panorama[creator] = new_obs;
+        let panorama_stakes = PanoramaStakes::new(&self.panorama, &self.weights);
+        info!(?panorama_stakes, "panorama stakes");
```

Comment:
With 100 validators that will be 100 INFO-level logs per round; isn't that a bit spammy? Wouldn't it be enough to print it once per round?